pkg/config/config.go

```diff
@@ -37,37 +37,37 @@ var (
 )
 
 type ClusterConfig struct {
-	URL string `json:"url"`
-
-	ClusterCIDR          string `json:"clusterCIDR"`
-	ServiceCIDR          string `json:"serviceCIDR"`
-	ServiceNodePortRange string `json:"serviceNodePortRange"`
-	DNS                  string `json:"dns"`
-	Domain               string `json:"domain"`
-	MTU                  string `json:"mtu"`
+	// URL string `json:"url"`
+
+	// ClusterCIDR          string `json:"clusterCIDR"`
+	// ServiceCIDR          string `json:"serviceCIDR"`
+	// ServiceNodePortRange string `json:"serviceNodePortRange"`
+	// DNS string `json:"dns"`
+	// Domain               string `json:"domain"`
+	// MTU string `json:"mtu"`
 }
 
 type DebugConfig struct {
-	Pprof bool `json:"pprof"`
+	// Pprof bool `json:"pprof"`
 }
 
 type MicroshiftConfig struct {
-	ConfigFile string `json:"configFile"`
-	DataDir    string `json:"dataDir"`
-
-	AuditLogDir string `json:"auditLogDir"`
-	LogVLevel   int    `json:"logVLevel"`
-
-	Roles []string `json:"roles"`
-
-	NodeName string `json:"nodeName"`
-	NodeIP   string `json:"nodeIP"`
-
-	Cluster ClusterConfig `json:"cluster"`
-
-	Manifests []string `json:"manifests"`
-
-	Debug DebugConfig `json:"debug"`
+	// ConfigFile string `json:"configFile"`
+	// DataDir    string `json:"dataDir"`
+	//
+	// AuditLogDir string `json:"auditLogDir"`
```

Comment:
Is the proposal to log these to stdout instead?

Reply:
Keeping an on-disk layout consistent with OCP reduces our documentation, training, and tooling load.  It's not configurable on OCP, let's use the same locations.

log rotation is handled in-process by the kube-apiserver.

When the config.openshift.io types are brought in, there's an option for audit logging levels.  Disabling audit logging is accompanied with a warning that, "if something fails, it's quite likely we won't be able to find it without audit logs"